content/docs/tutorials/aws/ec2-webserver.md

```diff
@@ -55,66 +55,85 @@ aws:region: (us-east-1)
 
     ```javascript
     const aws = require("@pulumi/aws");
-
+    
```

Comment:
Extra spaces?

Plus looks like there are now two blank lines here - probably can be just one?